examples/with-redux-thunk/pages/index.js

```diff
@@ -22,9 +22,9 @@ const Index = () => {
 }
 
 export async function getStaticProps() {
```

Comment:
Can we remove SSG from this page 🙏 

There's usually no good use case to have the store in the server for static generation, it's better to return data and then hydrate the store, so ideally I would like to keep the main pages without data fetching methods and then add more pages like `/ssg` and `/ssr` to show the other usage.

Reply:
Done.